docs/src/pages/[platform]/theming/default-theme/web.mdx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { CardLink } from '@/components/CardLink';
+import { Grid, Icon } from '@aws-amplify/ui-react';
+import { ColorsIcon, TypographyIcon, SizesIcon } from './icons';
+import { MdOutlineAutoAwesome } from 'react-icons/md';
+
+Amplify UI follows the [System UI Theme Specification](https://system-ui.com/theme/) with some modifications. This allows for a consistent pattern when defining our default tokens for properties such as color, fontSizes, border radius, and more. In addition to the reference here, you can browse our [default tokens](https://github.com/aws-amplify/amplify-ui/tree/main/packages/ui/src/theme/tokens) and their corresponding types on Github.
```

Comment:
The System UI link appears to be broken for me. 
nit: "fontSizes" => "font size"
nit: "Github" => "GitHub"